dask/imperative.py

```diff
@@ -183,6 +183,25 @@ def get(dsk, keys, get=None, **kwargs):
     return get(dsk2, keys, **kwargs)
 
 
+def compute(*args, **kwargs):
+    """Evaluate several ``Value``s at once.
+
+    Examples
+    --------
+    >>> a = value(1)
+    >>> b = a + 2
+    >>> c = a + 3
+    >>> compute(b, c)  # Compute both simultaneously
+    (3, 4)
+    """
+    if len(args) == 1:
+        return args[0].compute(**kwargs)
```

Comment:
Recommend either always returning a tuple or changing the other `compute` functions to have this policy.